platform/src/main/java/platform/pageobjects/Footer/FooterPage_PT90_106.java

```diff
@@ -0,0 +1,723 @@
+
+		CommonFunctions.scrolltoElement(withdraw);
+		if (CommonFunctions.isElementClickable(withdraw)) {
+			withdraw.click();
+			log.info("Withdraw Link working");
+		} else {
+			log.error("Withdraw Link is not working");
+		}
+	}
+
+	public void clickLoginFooterLink() throws Exception {
+
+		CommonFunctions.scrolltoElement(footerLogin);
+		if (CommonFunctions.isElementClickable(footerLogin)) {
+			footerLogin.click();
+			log.info("Footer Login Link working");
+		} else {
+			log.error("Footer Login Link is not working");
+		}
+	}
+
+	public void clickRegistrationFooterLink() throws Exception {
+
+		CommonFunctions.scrolltoElement(footerRegister);
+		if (CommonFunctions.isElementClickable(footerRegister)) {
+			footerRegister.click();
+			log.info("Footer Registration Link working");
+		} else {
+			log.error("Footer Registration Link is not working");
+		}
+	}
+
+	public void clickTermsofuseForLinksValidation() throws Exception {
+
+		CommonFunctions.scrolltoElement(termsOfUseLink);
+		if (CommonFunctions.isElementClickable(termsOfUseLink)) {
+			termsOfUseLink.click();
+			log.info("Terms of User Footer Link working");
+		} else {
+			log.error("Terms of User Footer Link working");
+		}
+
+	}
+
+	// Testcase no: PT-102 to PT-105
+	public void gotoOrderStepOneForFooterValidation() throws Exception {
+
+		CommonFunctions.waitForVisiblity(productListLink, waitTime);
+		productListLink.click();
+		log.info("Click on product list page");
+
+		CommonFunctions.waitForVisiblity(selectKanjiProductFromProductList, waitTime);
+		selectKanjiProductFromProductList.click();
+		log.info("Click on buy button from product list page,select 1st product");
+
+	}
+
+	// Testcase no: PT-102 to PT-105
+	public void gotoOrderStepTwoForFooterValidation() throws Exception {
+		CommonFunctions.waitForVisiblity(productListLink, waitTime);
+		productListLink.click();
+		log.info("Click on product list page");
+
+		CommonFunctions.waitForVisiblity(selectKanjiProductFromProductList, waitTime);
+		selectKanjiProductFromProductList.click();
+		log.info("Click on buy button from product list page,select 1st product");
+
+		CommonFunctions.isElementVisible(step2ProceedButton);
+		CommonFunctions.waitForVisiblity(step2ProceedButton, waitTime);
+		step2ProceedButton.click();
+
+		NewBaseClass orderflowauth = new NewBaseClass();
+		orderflowauth.replaceurl();
+
+		log.info("go to relogin page");
+		LoginPage loginObj = new LoginPage(driver);
+		loginObj.loginToPlatformForPayment();
+
+	}
+
+	// Testcase no: PT-102 to PT-105
+	public void gotoOrderStepThreeForFooterValidation() throws Exception {
+
+		CommonFunctions.isElementVisible(step3ProceedButton);
+		CommonFunctions.waitForVisiblity(step3ProceedButton, waitTime);
+		step3ProceedButton.click();
+
+	}
+
+	/*
+	 * public void clickFooterScbtLink() {
+	 * 
+	 * CommonFunctions.scrolltoElement(footerScbtLink); if
+	 * (CommonFunctions.isElementClickable(footerScbtLink)) {
+	 * footerScbtLink.click(); }
+	 * 
+	 * }
+	 */
```

Comment:
Please remove commented code, if you are never planning to use it

Reply:
Done the requested changes.